src/main/java/pl/sda/weather/repository/ILocationRepository.java

```diff
@@ -0,0 +1,18 @@
+package pl.sda.weather.repository;
+
+
+import pl.sda.weather.model.LocationModel;
+
+import java.util.List;
+
+public interface ILocationRepository {
+
+
+    void addLocationModelToTxtDB(LocationModel locationModel);
+
+    void addLocationModelJsonToDB(LocationModel locationModel);
+
+    List<LocationModel> getLocationModelFromFileTxt();
```

Comment:
idea behind interface is to have only one method :)

```
List<LocationModel> getLocationModelFromFileTxt();
List<LocationModel> getLocationModelFromFileJson();
```

It's the same method and should have two implementations in LocationTxtRepository and LocationJsonRepository